### PR TITLE
IPE-513: Add Dockerfile

### PR DESCRIPTION
**Pull request by ecp88**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+# copy your source tree
+COPY ./src ./src
+
+# build for release
+RUN cargo build --release
+
+FROM ubuntu:20.04
+RUN apt update && apt install -yy openssl ca-certificates
+COPY --from=builder /tmp/target/release/indexer-accounts .
```
Comment (telezhnaya) [inline Dockerfile:17]: Looks like you copy pasted from the other project and forgot to change the names

**Review by telezhnaya — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (cejas-ridian) [reply Dockerfile:17]: `indexer-accounts` is the name of the package? https://github.com/near/near-indexer-events/blob/main/Cargo.toml#L2

Comment (ecp88) [reply Dockerfile:17]: `indexer-accounts` is the name of the package? https://github.com/near/near-indexer-events/blob/main/Cargo.toml#L2

Comment (telezhnaya) [reply Dockerfile:17]: lol, thank you
That was my copy-paste error
I'll merge this, fix the names and send the PR with you as the reviewer

**Pull request merged**